packages/metrics-app/.eslintrc.json

```diff
@@ -3,6 +3,7 @@
   "rules": {
     "unicorn/consistent-function-scoping": "off",
     "@typescript-eslint/member-ordering": "off",
-    "sort-keys": "off"
+    "sort-keys": "off",
+    "import/newline-after-import": [ "error", { "count": 2 }]
```

Comment:
Is this eslint rule necessary? I think it might be included in the eslint preset already, but I might be wrong.